lib/cassandra_migrations/cassandra/keyspace_operations.rb

```diff
@@ -7,16 +7,18 @@ module KeyspaceOperations
 
       def create_keyspace!(env)
         config = Config.configurations[env]
+        validate_config(config)
+
+        execute(
+          "CREATE KEYSPACE #{config.keyspace} \
+           WITH replication = { \
+             'class':'#{config.replication['class']}', \
+             'replication_factor': #{config.replication['replication_factor']} \
+           }"
+        )
```

Comment:
That totally makes sense. I didn't explore the keyspace creation much while I worked on this.